### PR TITLE
Bugfix: Scheduler setting conflict.

### PR DESCRIPTION
**Pull request by zhang-ge-hao**
The scheduler was set in [`sft_qwencoder.sh`](https://github.com/QwenLM/Qwen2.5-Coder/blob/186f7370bb70363b100f1b812026703a43240b27/finetuning/sft/scripts/sft_qwencoder.sh#L66).

The schedular **also** was set in [`default_offload_opt_param.json`](https://github.com/QwenLM/Qwen2.5-Coder/blob/186f7370bb70363b100f1b812026703a43240b27/finetuning/sft/configs/default_offload_opt_param.json#L14-L21).

This conflict made some weird behaviors. For example, when running warmup+cosine decay as the `sft_qwencoder.sh` originally set, people expected to get the learning rate like this:

a682830d31c377d0eef8268d7a0798ed

However, here is what I got:

ec69bc8941c9cc3fb8d28827a9b8ef09

The learning rate warmup has followed some curves and decayed linearly.

After this PR's modification, i.e., removing the config on the DeepSpeed side, the learning rate schedular worked as expected.

image

### CODE REVIEW

**Review by CSJianYang — APPROVED** [review]

**Comment by CSJianYang** [comment]
Remove scheduler setting in Deepspeed config

**Comment by CSJianYang** [comment]
Bugfix: Scheduler setting conflict.
